bin/run-librivox.sh

```diff
@@ -5,10 +5,26 @@ if [ ! -f DeepSpeech.py ]; then
     exit 1
 fi;
 
+if [ ! -d "${ds_dataroot}" ]; then
+    ds_dataroot = "data"
+fi;
 
+# Warn if we can't find the train files
+if [ ! -f "${ds_dataroot}/librivox-train-clean-100.csv" ]; then
+    echo "Warning: It looks like you don't have the LibriSpeech corpus "       \
+         "downloaded and preprocessed. Make sure \$ds_dataroot points to the " \
+         "folder where the LibriSpeech data is located, and that you ran the " \
+         "importer script before running this script."
+fi;
```

Comment:
I agree on adding the file name of the importer script here. It'll save us many a random issue filed by contributors.